esports_data_from_pandascore/README.md

```diff
@@ -102,6 +74,12 @@ different match you update the `match_id` value to another ID.
 If you haven't already as part of the [Setup](#setup), go ahead and replace
 the `api_key` value with your PandaScore API key.
 
+> PandaScore API [Terms and Conditions](https://pandascore.co/terms-and-condition)
+> do not allow us to share their API endpoints publicly. To use this example,
+> you need to replace `matches_api_endpoint` in [`fn-call.json`](./fn-call.json)
+> with the PandaScore API matches endpoint, which you can find on their
+> [List marches API call documentation page](https://developers.pandascore.co/reference/get_matches).
```

Comment:
After we added request size limit, I ran into that limit when trying to scrub their documentation page for the API endpoint. I decided to remove that functionality (and the quick start which needed it in place) and now I'm just asking the user to pass that API endpoint as a parameter. I think that's fine for now and at least the code is clean.